packages/reports/app/styles/navi-reports/components/grouped-list.less

```diff
@@ -8,35 +8,24 @@
 
   &__group-header {
     cursor: pointer;
-    margin: 2px 0;
+    margin: 3px 0;
```

Comment:
not a blocker - should we use a regular cursor if we're in search/show selected mode? clicking the header doesn't collapse the group

Reply:
Good call, this has some weird behavior. The previous behavior was to always open or close (so even when searching/show selected you could close the group), the current behavior affects the state on the component, but is overridden by the search/show selected (so if you click during a search, then clear the search you will see the group opened).

I'm thinking it might be worth opening an issue for this (also adding `expand all`/`collapse all`)

Reply:
https://github.com/yahoo/navi/issues/681